src/time/span.cr

```diff
@@ -84,70 +95,46 @@ struct Time::Span
     @nanoseconds = nanoseconds.to_i32
   end
 
+  # Creates a new `Time::Span` from the *nanosenconds* given
+  #
+  # Nanoseconds get normalized in the range of 0...1_000_000_000,
```

Comment:
```suggestion
  # Nanoseconds get normalized in the range of `0...1_000_000_000`,
```

Reply:
Thanks, sorry for wasting your time 😓

Reply:
Don't worry. Thanks to you for your contribution!